docs/reference/discovery.mdx

```diff
@@ -14,10 +14,12 @@ There are two ways an app can use Discovery:
 
 When authenticating via FCL using Discovery UI, a user is shown a list of services they can use to login.
 
-![FCL Default Discovery UI](../images/discovery-ui.jpg)
+![FCL Default Discovery UI](../images/discovery.jpg)
```

Comment:
This shows as a link. Where should it be linked? When I clicked the link in the preview it shows a 404.

Reply:
my bad - should link to `png`, not jpg